components/hab/src/command/pkg/channels.rs

```diff
@@ -41,11 +41,16 @@ use crate::{error::Result,
 /// # Failures
 ///
 /// * Fails if it cannot find the specified package in Builder.
-pub fn start(ui: &mut UI, bldr_url: &str, ident: &PackageIdent, token: Option<&str>) -> Result<()> {
+pub fn start(ui: &mut UI,
+             bldr_url: &str,
+             ident: &PackageIdent,
+             target: PackageTarget,
+             token: Option<&str>)
+             -> Result<()> {
     let api_client = Client::new(bldr_url, PRODUCT, VERSION, None)?;
 
-    ui.begin(format!("Retrieving channels for {}", ident))?;
-    let channels = api_client.package_channels(ident, token)?;
+    ui.begin(format!("Retrieving channels for {} ({})", ident, target))?;
```

Comment:
I think I prefer the parenthesized target that you have here over something like 

```rust
debug!("Demoting package {}, target {}", ident, target);
```
elsewhere.

Reply:
Yeah I tried it out and changed it from (target: {}) to just ({})

Leaving the debug statements as is for now seems ok

Reply:
I just realized that this is a `ui.begin` call, whereas everything else was a `debug` call, and thus not visible by default.

Given that we've never emphasized target like this before, I'm a little concerned about the lack of feedback that's visible by default. 

In particular, I'm concerned about a user, say, demoting a linux2-targeted package that happens to have the same identifier as a linux-target package (or any such combination of targets with identical identifiers), but forgetting that they're defaulted to a linux target and demoting the wrong package, with no obvious feedback that that's what happened.

Reply:
Are you proposing additional output lines or something else @christophermaier ?

Reply:
Additional output lines.

Reply:
👍 will add